infra.bs

```diff
@@ -290,8 +296,8 @@ the <a>string</a> with their corresponding <a>code point</a> in <a>ASCII upper a
 
 <hr>
 
-<p>To <dfn export>strip newlines</dfn> from a <a>string</a>, remove any U+000A LINE FEED and
-U+000D CARRIAGE RETURN <a>code points</a> from the <a>string</a>.
+<p>To <dfn export>strip newlines</dfn> from a <a>string</a>, remove any U+000A LF and U+000D CR
```

Comment:
I think you can use either per http://ftp.unicode.org/Public/UNIDATA/NamesList.txt. Also, I guess you meant to reference http://www.unicode.org/charts/PDF/U0000.pdf?